draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,129 @@
+  \item \textbf{2} × Near (IV-V) Map Tile,
+  \item \textbf{1} × Center (VI-VI) Map Tiles containing Settlements and Grails (Only for 4 players).
+\end{itemize}
+\textbf{AND}
+
+\begin{itemize}
+  \item \textbf{1} × Starting (I) Map Tile - with \textit{Dragon Utopia} for 2-3 players \textbf{OR}
+  \item \textbf{2} × Starting (I) Map Tile - with \textit{Dragon Utopia} for 4 players.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+\textit{The game ends at the end of the Round where the first victory condition is met:}
+\begin{itemize}
+  \item One player has defeated all other players’ main Heroes once - \textit{That player wins the game.}
+  \item One player conquers both of the Dragon Utopias - \textit{That player wins the game.}
+  \item One player controls 5 mines or settlements on Near and Center Tiles - \textit{That player gets 4VP.}
+  \item At the end of Round 13, \textit{or at the end of Round 8 for the short game.}
+\end{itemize}
+
+\subsection*{\MakeUppercase{The player with the most Victory Points wins the game:}}
+\begin{itemize}
+  \item 5 VP for conquering a Dragon Utopia
+  \item 4 VP for defeating the Enemy’s Main Hero - \textit{once per defeated Faction.}
+  \item 2 VP for defeating the Enemy’s Secondary Hero - \textit{once per defeated Faction.}
+  \item 2 VP for each enemy Main Town captured - \textit{once per captured Faction.}
+  \item 1 VP for every controlled Mine or Settlement.
+  \item 1 VP for every Level of Experience of the Main Hero.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\begin{itemize}
+  \item Start of \textbf{\nth{1} and \nth{9} Rounds:} All Heroes gain +1 Movement.
+  \item Start of \textbf{\nth{4} and \nth{8} Rounds:} Remove black Cubes from all locations that give resources, morale or Cards.
+  \item End of \textbf{\nth{10} Round:} Player(s) with lowest XP roll 2 Resource dice.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+  \item Remove “Earthquake”, “Remove Obstacles”, “View Earth” and “Visions" from the spell Deck.
+  \item Remove “Unexpected Reinforcements” from the Astrologers Proclaim Deck.
+  \item Make one Deck of minor, one of major and one of relic artifacts.
+  \item Make one Deck of basic spells and one of expert spells.
+  \item On Starting and Far Tiles players can only get basic spells and minor artifacts.
+  \item Players may not use Diplomacy to skip lvl VII Combat.
+  \item If your Hero is lvl VII, treat lvl VI neutrals as lvl VII and gain 10 \svg{gold}
+  \item Sanctuary effect: +1 Morale and draw 1 Card. Visitable once per Faction.
+  \item Obelisk effect: Roll one Resource die and draw one Card. \textit{May be visited once per Faction.}
+  \item When fighting a neutral Combat, a player may add +1 difficulty for an extra 5 \svg{gold} + Search (2) the Artifact or the Spell Deck
+  \item Level VII settlements are treated as two settlements for rewards AND roll one Resource or Treasure (reroll any XP) die.
```

Comment:
```suggestion
  \item Settlements on Center (VI-VII) Map Tiles are treated as two Settlements in terms of rewards AND allow rolling one \svg{resource} or \svg{treasure} (reroll any XP).
```
By two, you mean treat them as two separate settlements? You can turn it into two separate mines? Or one mine and reinforcing one unit for free? Also, there's only one such settlement in the game (Tower on tile #C1).